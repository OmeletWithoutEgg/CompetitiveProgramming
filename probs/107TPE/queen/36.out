-470981.21 -37404247.53
