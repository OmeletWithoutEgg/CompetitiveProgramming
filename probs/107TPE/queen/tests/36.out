-9328802.66 4703308.46
